heka/files/toml/output/aggregator.toml

```diff
@@ -0,0 +1,7 @@
+{%- extends "heka/files/toml/output/tcp.toml" %}
+{%- block address -%}
+address = "{{ output.host }}:5565"
```

Comment:
I changed my mind. I'd like to do it with the refactoring that will move the output configuration to the Heka formula `heka.yml` support metadata.